src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -232,9 +238,12 @@ def get_aggregate_prediction_intervals(
             modeled_bounds = pd.concat([modeled_bounds, remaining_bounds_w_models])
 
         # construction conformal corrections using Gaussian models
-        # get means and standard deviations for for aggregates
+        # get means and standard deviations for aggregates
         # and add correction (which is percentile of the Gaussian at the quantile we care about)
+        self.modeled_bounds_agg = modeled_bounds
+        self.conformalization_data_agg = conformalization_data
```

Comment:
same here, defined as `None` in the constructor

Reply:
Same, done